classes/tank.class.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+class Tank{
+    public $life;
```

Comment:
I think here you need to put everything with private because we can have a problem later.

Reply:
Private or protected? Wich do you think is better ? @RobertPlaiasu  @GrigoAleex